packages/cadl-vscode/scripts/dogfood.js

```diff
@@ -2,7 +2,7 @@
 import { run } from "@cadl-lang/internal-build-utils";
 import { readFileSync } from "fs";
 
-const version = JSON.parse(readFileSync("package.json").toString()).version;
+const version = JSON.parse(readFileSync("package.json", { encoding: "utf-8" })).version;
```

Comment:
why do you need to do that? is that something we should do more often?

Reply:
There was a ts-check typing error elsewhere, then I fixed it likewise here for consistency in the same style. I've now gone through and used "utf-8" arg throughout as we already had in a lot of places. In theory this seems like it would give readFile API the most opportunity to optimize as it is not required to produce an intermediate Buffer. I don't know if this is really better in practice, but it looks a little nicer to me.